### PR TITLE
Add Containers for Change

### PR DESCRIPTION
**Pull request by Scottie0001**
Add Containers for Change in Queensland and Western Australia

### CODE REVIEW

**Comment by AntiCompositeNumber** [comment]
It looks like the Queensland and the WA programs use the same brand, but with different operators (Container Exchange and Western Australia Return Recycle Renew Limited, respectively). They have the same logo, same website, and same telephone shortcode. I think it would be better to use the same `brand=` and `brand:wikidata=` for both, but use separate `operator=` and `operator:wikidata=` tags for the two programs.

**Comment by Scottie0001** [comment]
Each return point is operated by different companies. The Containers for Change program is managed by a government organisation in each state.

File: data/brands/amenity/recycling.json [hunk 1]
```diff
@@ -92,6 +92,38 @@
...
       }
     },
+    {
+      "displayName": "Containers for Change (Queensland)",
+      "id": "",
+      "locationSet": {"include": ["au-qld.geojson"]},
+      "preserveTags": ["^name"],
+      "tags": {
+        "amenity": "recycling",
+        "brand": "Containers for Change (Queensland)",
```
Comment (AntiCompositeNumber) [inline data/brands/amenity/recycling.json:102]:
```suggestion
        "brand": "Containers for Change",
```

File: data/brands/amenity/recycling.json [hunk 2]
```diff
@@ -92,6 +92,38 @@
...
+      }
+    },
+    {
+      "displayName": "Containers for Change (Western Australia)",
+      "id": "",
+      "locationSet": {"include": ["au-wa.geojson"]},
+      "preserveTags": ["^name"],
+      "tags": {
+        "amenity": "recycling",
+        "brand": "Containers for Change (Western Australia)",
```
Comment (AntiCompositeNumber) [inline data/brands/amenity/recycling.json:118]:
```suggestion
        "brand": "Containers for Change",
```

**Review by AntiCompositeNumber — COMMENTED** [review]
In that case, let's just remove the disambiguators from `brand=`.

**Comment by bhousel** [comment]
Yeah this looks like just adding 2 of the same thing..  If they had different operators then it would make sense, but these items don't have `operator` tags anyway.  I'll just merge and combine.

**Comment by bhousel** [comment]
update: a Wikidata admin removed [Q124060545](https://www.wikidata.org/wiki/Special:Log?page=Q124060545) in April, so it sounds like they also agree that there doesn't need to be multiple entries for this brand.

**Pull request merged**